folly/Format-inl.h

```diff
@@ -774,9 +774,7 @@ class TryFormatValue {
 
 template <class T>
 class TryFormatValue<
-  T,
-  typename std::enable_if<
-      0 < sizeof(FormatValue<typename std::decay<T>::type>)>::type>
+  T, typename FormatValue<typename std::decay<T>::type>::type>
```

Comment:
I'm not certain,  but I believe this is due to MSVC not yet supporting expression SFINAE. Whatever the actual reason MSVC doesn't like the sizeof on an invalid type.